backend/src/data/models/interview/interview.model.ts

```diff
@@ -15,7 +15,7 @@ class Interview extends Abstract {
 
   public 'intervieweeUserId': number;
 
-  public 'interviewerUserId': number;
+  public 'interviewerUserId': number | null;
```

Comment:
Have you checked if `interviewer_user_id` is nullable inside `interviews` table?

Reply:
Yes, this filed is nullable
<img width="408" alt="image" src="https://user-images.githubusercontent.com/56650388/189375387-3a3d9280-a540-405f-bc1c-d91fbf7dc704.png">

Reply:
Okay, just thought about `20220826130346_update_interviews_constraints.ts` migration where `not nullable` constraint has been added for `interviewee_user_id` 😕